hikari/interactions/command_interactions.py

```diff
@@ -145,6 +145,12 @@ class CommandInteraction(base_interactions.MessageResponseMixin[CommandResponseT
     This will be `builtins.None` for command interactions triggered in DMs.
     """
 
+    guild_locale: typing.Optional[str] = attr.field(eq=False, hash=False, repr=True)
+    """The preferred language of the guild this command interaction was triggered in.
```

Comment:
Can we document the fact that guild_locale defaults to en-us and can only be set by community guilds here as a note section, IMO this is pretty important info to communicate